src/EnergyPlus/TARCOGGasses90.cc

```diff
@@ -159,7 +158,7 @@ namespace TARCOGGasses90 {
 		//xgrho = 0.0d0 //Autodesk:Uninit Force default initialization
 
 		//Simon: remove this when assigned properly
-//		grho = 0.0; //Unused
+		//		grho = 0.0; //Unused
```

Comment:
Delete these two lines? And others later on.  it's not a huge deal if it's useful to keep these around for clarity, except these stick out now because the // got aligned but not there's extra space after the //.